src/views/Product/Page.tsx

```diff
@@ -1,65 +1,100 @@
 import { smallScreen } from "../../globalStyles/scss/variables.scss";
 
 import classNames from "classnames";
-import * as React from "react";
+import { isEmpty } from "lodash";
+import { History } from "history";
```

Comment:
I'm not sure if this is a good layer to handle routing, personally I'd move it to the view